utils/lib/nr-graphql-helpers.ts

```diff
@@ -124,10 +124,22 @@ export const translateMutationErrors = (
   }
 };
 
+type CategoryTermsNRGraphqlResults = {
+  actor: {
+    nr1Catalog: {
+      categories: {
+        displayName: string;
+        slug: string;
```

Comment:
Are these fields even needed? On lines 165-167 we are just flat mapping down to the terms.

Reply:
We do not - should i update the query to reflect this as well?

Reply:
updated on the typescript side of things 👍🏻

Reply:
Might as well remove those from the query as well, no need to request more than we need!